apps/docs/content/tokens/semantic/color.mdx

```diff
@@ -12,10 +12,23 @@ export const tabs = [
   { title: "Dark", category: categoryKey }
 ];
 
+# Color
+
+## Guidelines
+
+Colours are one of the most obvious way to convey meaning in design. Depending on its context of use and on the amount of time we see them, we start to associate colour with meaning. Each of the colours used in the Workleap ecosystem complements each other and have a specific role to play.
+
+### Communication is key
+
+Although we value an aesthetically pleasing use of colour, we place a higher value on clear communication. Colour supports the purpose of the content, communicating things like hierarchy, interactive states and differences between visual elements.
+
+### Colours have meaning
+
+Each colour have assigned sentiment, based on how they function within the interface. Defined colour roles make things easy to modify and customize later. Their meaning is also expanded to all Workleap verticals so that users understand that they’re in the same ecosystem and can recognize its codes.
```

Comment:
```suggestion
Each colour has assigned sentiment based on how they function within the interface. Defined colour roles make things easy to modify and customize later. Their meaning is also expanded to all Workleap verticals so that users understand that they’re in the same ecosystem and can recognize its codes.
```